docs/user.md

```diff
@@ -254,6 +254,23 @@ metadata:
 Note that timezone is required for `timestamp`. Otherwise, offset is relative
 to UTC, see [RFC 3339 section 5.6) 3339 section 5.6](https://www.ietf.org/rfc/rfc3339.txt).
 
+## Setting up a standby cluster
+
+Standby clusters are like normal cluster but they are streaming from a remote cluster. Patroni supports it, [read this](https://github.com/zalando/patroni/blob/bd2c54581abb42a7d3a3da551edf0b8732eefd27/docs/replica_bootstrap.rst#standby-cluster) to know more about them.
+
+As the first version of this feature, the only scenario covered by operator is to stream from a wal archive of the master. Following the more popular infrastructure of using Amazon's S3 buckets, it is mentioned  as s3_wal_path here. To make a cluster as standby add a section standby in the YAML file as follows.
+
+```yaml
+spec:
+  standby:
+    s3_wal_path: "s3 bucket path to the master"
+```
+Things to note:
+
+- An empty string is provided in s3_wal_path of the standby cluster will result in error and no statefulset will be created.
+- If standby is not needed, simply remove the standby_cluster section from the patroni through patronictl edit-config.
```

Comment:
this mean the standby leader will be promoted to the normal master or ?

Reply:
Let's remove this second part or make it an "active" sentence like:

To promote a standby cluster manually, use patronictl and remove config entry.